### PR TITLE
Update regularUsers

### PR DESCRIPTION
**Pull request by gromakovsky**
Problem: `hangyas` and `masiama` stopped working at Serokell.
We have new developers instead. We should change the list of
users who can access our servers.

Solution: replace former employees with current employees.

### CODE REVIEW

**Comment by balsoft** [comment]
FYI you can update just serokell.nix with `nix flake lock --update-input serokell-nix --commit-lock-file` in this repository, then push the changes here.

To deploy, you can either run `deploy` from `nix develop` yourself (since you should have wheel access everywhere now) or ask someone from ops to do so.

**Comment by notgne2** [comment]
`--skip-checks` will skip running flake checks, which you probably don't need, and will thus speed things up, `--auto-rollback false` stops the server from attempting to roll back if there's an error activating, `--magic-rollback false` turns of the system that rolls back the server if it becomes inaccessible to you (this is sometimes desirable if activation might take a long time and accidentally trip a rollback), and the last 2 flags are for ensuring it can use our Wasabi Nix cache.

So basically, they're both good options, just `deploy` will work fine, but you can mess around with it a bit if you want to.

**Comment by gromakovsky** [comment]
Aha, ok, thanks for the explanation. However, I am getting an error because I am trying to run it from MacOS:
```
error: a 'x86_64-linux' with features {} is required to build '/nix/store/210xqfa92riwakj36qpbg2xh36hdisnd-builder.pl.drv', but I am a 'aarch64-darwin' with features {benchmark, big-parallel, nixos-test}
🚀 ❌ [deploy] [ERROR] Failed to push profile: Nix build command resulted in a bad exit code: Some(1)
```

How hard is it to make it work on MacOS?

**Comment by balsoft** [comment]
Interesting that it didn't fetch it from cache, since Albali should've pre-built everything for you :thinking: 

Well, your best bet is probably to use albali as a remote builder

**Comment by notgne2** [comment]
I know next to nothing about cross compiling other than knowing it is possible sometimes, but you can also try configuring a remote builder, or specifying one ad-hoc with the `--builders` flag (after `--` which denotes the end of deploy-rs flags and the start of Nix flags)

**Comment by balsoft** [comment]
Ah, is wasabi down again?

**Comment by balsoft** [comment]
No, seems to be up, and the path is there.

@gromakovsky I think you just need to fix your binary cache access or something like that

**Comment by gromakovsky** [comment]
Okay, I'll try. I actually didn't get to setting up our binary cache yet, but I already have credentials, so I'll get to it soon.

**Comment by gromakovsky** [comment]
As we've [discussed in Slack](https://serokell.slack.com/archives/C742ZKYF5/p1652180541770179), setting up binary cache does not help me on MacOS. Can I SSH into some machine and do `deploy` inside `nix develop` there? Which machine should I use for that?

**Comment by gromakovsky** [comment]
Alright, I was able to do it inside `ssh albali.aquarius.serokell.team -p 17788 -A` where I cloned this repo, entered `nix-shell -p nixUnstable`, entered `nix develop` and ran `deploy`. Got this eventually:
```
⭐ ℹ️ [activate] [INFO] Activation succeeded!
⭐ ℹ️ [activate] [INFO] Magic rollback is enabled, setting up confirmation hook...
👀 ℹ️ [wait] [INFO] Found canary file, done waiting!
⭐ ℹ️ [activate] [INFO] Waiting for confirmation event...
🚀 ℹ️ [deploy] [INFO] Success activating, attempting to confirm activation
🚀 ℹ️ [deploy] [INFO] Deployment confirmed.
```
